draft-ietf-trans-rfc6962-bis.md

```diff
@@ -637,16 +637,22 @@ following requirements:
   that the Transparency Information ({{x509v3_transinfo_extension}}) extension
   MUST be omitted.
 
-* `SignedData.signerInfos` MUST contain a signature from the same (root or
-  intermediate) CA that will ultimately issue the certificate. This signature
-  indicates the CA's intent to issue the certificate. This intent is considered
-  binding (i.e., misissuance of the precertificate is considered equivalent to
-  misissuance of the certificate). (Note that, because of the structure of CMS,
-  the signature on the CMS object will not be a valid X.509v3 signature and so
-  cannot be used to construct a certificate from the precertificate).
+* `SignedData.signerInfos` MUST contain 1 `SignerInfo` with a `signature` from
```

Comment:
I think we do - the part on the log side doesn't have to be a fully-compliant RFC5652 receiver, I think there are plenty of other attributes it will ignore.

We can specify that the sender (producer of the CMS precert) must use the subjectKeyIdentifier approach and no other signedAttrs on unsignedAttrs, so that means the implementation on the log side doesn't have to care about any of those other than what's specified here.